docs/devsite-help/client-configuration.md

```diff
@@ -23,6 +23,12 @@ and `Google.Cloud.Translation.V2` libraries support builders as well, but with f
 
 ## Specifying credentials
 
+**Important**: If you accept a credential configuration (credential JSON/File/Stream) from an external source
```

Comment:
is there going to be another PR for adding the warning in the method's documentation?

Reply:
Yes, but that's on https://github.com/googleapis/google-api-dotnet-client.

Reply:
For completion: https://github.com/googleapis/google-api-dotnet-client/pull/2916